README.md

```diff
@@ -28,38 +28,48 @@ The npm installs the binaries as both `raml-toolkit` and `ramlint` and they can
 **Note:** Some commands require environment variables to be set. This can be done using a [.env file](https://www.npmjs.com/package/dotenv#rules) in your working directory (the directory from which you run `raml-toolkit`).
 
 <!-- commands -->
-- [`raml-toolkit diff APISPECBASEPATH APISPECNEWPATH`](#raml-toolkit-diff-apispecbasepath-apispecnewpath)
+- [`raml-toolkit diff BASE NEW`](#raml-toolkit-diff-base-new)
 - [`raml-toolkit download`](#raml-toolkit-download)
 - [`raml-toolkit lint [FILENAME]`](#raml-toolkit-lint-filename)
 
-#### `raml-toolkit diff APISPECBASEPATH APISPECNEWPATH`
+#### `raml-toolkit diff BASE NEW`
 
-Takes two API spec files as input and outputs the differences.
+This command has three modes: ruleset, diff-only, and directory.
 
 ```txt
 USAGE
-  $ raml-toolkit diff APISPECBASEPATH APISPECNEWPATH
+  $ raml-toolkit diff BASE NEW
 
 ARGUMENTS
-  APISPECBASEPATH  The base API spec file for the comparison
-  APISPECNEWPATH   The new version of the API spec for comparison against the base version
+  BASE  The base API spec file (ruleset / diff-only mode) or configuration (directory mode)
+  NEW   The new API spec file (ruleset / diff-only mode) or configuration (directory mode)
 
 OPTIONS
-  -h, --help             show CLI help
-  -r, --ruleset=ruleset  Path to ruleset to apply to diff
-  -v, --version          show CLI version
+  -h, --help               show CLI help
```

Comment:
Can you please make all the descriptions for options uppercase?

Reply:
Capitalized...not uppercase please.

Reply:
😂 you know what I meant

Reply:
Oh, this makes more sense. I saw the email notifications and thought @gkhunger-sf was having a conversation with himself.

Also, no. Those are oclif defaults. (Technically, yes, but I have an idea that's outside the scope of this PR.)